ci-operator/jobs/openshift/release/openshift-release-release-4.5-periodics.yaml

```diff
@@ -6476,6 +6476,9 @@ periodics:
     job-release: "4.5"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: release-openshift-ocp-installer-cluster-logging-operator-e2e-4.5
+  reporter_config:
+    slack:
+      channel: '#forum-logging'
```

Comment:
@stevekuznetsov thanks for setting this up, it looks like it worked:
https://coreos.slack.com/archives/CB3HXM2QK/p1588272718264500

but is there any way for us to customization the message?  minimally i'd like to tag the logging team alias so it gets their attention.

cc @jcantrill @alanconway

Reply:
actually i guess that was just a message that the job was triggered.  I see a few for job ending in success but i don't see any for the job ending in failure as it did here:
https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/logs/release-openshift-ocp-installer-cluster-logging-operator-e2e-4.5/15

Reply:
@bparees yeah, this is a fine chance for you to verify your go templating knowledge: https://github.com/openshift/release/blob/ff4f21d6488ee678eff94f8b3bc50213e272829f/core-services/prow/02_config/_config.yaml#L528

Reply:
@bparees today its not possible to specify a custom template or message on the job, but if that is important to you you could create a feature request for us